packages/node_modules/@webex/plugin-meetings/test/unit/spec/member/index.js

```diff
@@ -6,18 +6,12 @@ import Member from '@webex/plugin-meetings/src/member';
 
 
 describe('member', () => {
-  const sandbox = sinon.createSandbox();
-
-  afterEach(() => {
-    sandbox.restore();
-  });
```

Comment:
issue: You still need to do a `sinon.restore()` (instead of `sandbox.restore()`) in an `afterEach`